modules/sugar/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@ephox/sugar",
-  "version": "9.2.1",
+  "version": "9.3.0",
```

This is done at release time for the `develop` branch. What we need to do instead is add Sugar to the versions.txt file
```suggestion
  "version": "9.2.1",
```